CONTRIBUTING.md

```diff
@@ -19,7 +19,8 @@ You need to install the following dependencies:
   * Linux or macOS. Windows support is coming soon. <!-- TODO: Link Java roadmap when it is published -->
   * [JDK 1.8+](http://jdk.java.net/10/).
   * [Maven 3.5+](https://maven.apache.org/download.cgi).
-  * [Stable Rust](https://www.rust-lang.org/).
+  * [Rust 1.32.0](https://www.rust-lang.org/).
+  To install a specific Rust version, use `rustup install 1.32.0` command.
   * The [system dependencies](https://exonum.com/doc/version/0.10/get-started/install/) of Exonum. 
```

Comment:
Would it be better to use an abstract `VERSION` here?

Reply:
I think it is OK.

We might even keep both instructions (one commented out) because we need to switch between the two rather frequently.